Gemfile

```diff
@@ -1,31 +1,31 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "3.2.2"
+ruby '3.2.2'
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.0.8"
+gem 'rails', '~> 7.0.8'
 
 # The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
-gem "sprockets-rails"
+gem 'sprockets-rails'
 
 # Use sqlite3 as the database for Active Record
-gem "sqlite3", "~> 1.4"
+gem 'sqlite3', '~> 1.4'
 
```

Comment:
| Project Requirements 📝 |
|-|
| ![image](https://github.com/abdozayan12/DataApp/assets/93039359/b39ad88d-5789-43c6-9799-255633ed77e8) |

- [ ] Great work on making the migrations 👍 but i guess you missed one requirements 😨 that was to setup PostgreSQL in your rails application 🤔 but i saw that you have set up SQLite 👀 Kindly setup PostgreSQL instead of SQLite to follow the project requirements 😊